apps/api/src/domain/models/bibliographic-reference/types/CreatorType.ts

```diff
@@ -0,0 +1,5 @@
+export enum CreatorType {
+    author = 'Author',
```

Comment:
Let's not capitalize here. We can use a capitalization pipe in our presentational layer if the presenter intends to use capital, but under no circumstances should the user be required to 'uncapitalize' upsteram.

Reply:
Done